composer.json

```diff
@@ -18,7 +18,8 @@
     "illuminate/support": "5.5.x|5.6.x|5.7.x|5.8.x",
     "illuminate/translation": "5.5.x|5.6.x|5.7.x|5.8.x",
     "symfony/finder": "~3.0|~4.0",
-    "tanmuhittin/laravel-google-translate": "^0.11.0"
+    "tanmuhittin/laravel-google-translate": "^0.11.0",
+    "andrey-helldar/laravel-lang-publisher": "^1.0|^1.5"
```

Comment:
You can simply specify version `^1.0`.

The `^` operator behaves very similarly but it sticks closer to semantic versioning, and will always allow non-breaking updates. For example `^1.2.3` is equivalent to `>=1.2.3 <2.0.0` as none of the releases until 2.0 should break backwards compatibility.
[Source](https://getcomposer.org/doc/articles/versions.md#caret-version-range-)

Reply:
My bad. I did not checked your last version constraints. Looks like Version 1.5.4 -> https://packagist.org/packages/andrey-helldar/laravel-lang-publisher#v1.5.4 is compatible enough with this package. It supports Laravel 5.5+ just like this package.

Reply:
No errors 😊

Simply semantic versioning via `^` says that the composer can set the version of the package within `1.0.0 <= x < 2.0.0`.

If you specify version `^1.0`, then when installing dependencies, the compiler automatically finds the version of the package you need.

The caveat is that `Laravel 5.5` can work with `php 7.0` and higher, and version `^1.5` of the package only works with `php 7.1.3` and above, although Laravel 5.5 also supports it.

In simple terms, it is sufficient to specify version `^1.0`.